src/test/java/io/github/christophermanahan/carnitas/ResponseBuilderTest.java

```diff
@@ -0,0 +1,41 @@
+package io.github.christophermanahan.carnitas;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ResponseBuilderTest {
+    @Test
+    void itCanBuildAnHTTPResponse() {
+        HTTPResponse.Status status = HTTPResponse.Status.OK;
+        String contentLength = Headers.contentLength(0);
+        String allow = Headers.allow(List.of(HTTPRequest.Method.GET));
+        List<String> headers = List.of(contentLength, allow);
+        Optional<String> body = Optional.of("name=<something>");
+        ResponseBuilder builder = new ResponseBuilder();
+
+        HTTPResponse response = builder.setStatus(status)
+          .addHeader(contentLength)
+          .addHeader(allow)
+          .setBody(body)
+          .get();
+
+        HTTPResponse expectedResponse = new HTTPResponse(status, headers, body);
+        assertTrue(expectedResponse.equals(response));
+    }
+
+    @Test
+    void itCanBuildAnHTTPResponseWithOptionalHeadersAndBody() {
+        HTTPResponse.Status status = HTTPResponse.Status.OK;
+        ResponseBuilder builder = new ResponseBuilder();
+
+        HTTPResponse response = builder.setStatus(status)
+          .get();
+
+        HTTPResponse expectedResponse = new HTTPResponse(status, List.of(), Optional.empty());
+        assertTrue(expectedResponse.equals(response));
+    }
```

Comment:
Can I build an `HTTPResponse` without a status?

Reply:
@damonkelley Would an exception be an appropriate way to enforce not calling `get` before setting the status? Or would a default status be more appropriate? Or some other technique?

Reply:
Went with the default status of 200 OK for now.